6.x/base-how-to.md

```diff
@@ -413,3 +413,71 @@ Add whatever validation rules & inputs you want, in addition to name and passwor
 ```
 This will make the registration process pick up a view you can create, in ```resources/views/vendor/backpack/ui/auth/register.blade.php```. You can copy-paste the original view, and modify as you please. Including adding your own custom inputs.
 
+<a name="enable-email-verification-in-backpack-routes"></a>
+### Enable email verification in Backpack routes
+
+In Backpack CRUD 6.2 we introduced the ability to require email verification when accessing Backpack routes.
+
+To enable this feature please do the following: 
+
+**Step 1** - Ensure your Authenticable model (usually `App\Models\User`) implements the `Illuminate\Contracts\Auth\MustVerifyEmail` contract.
+
+```php
+<?php
+ 
+namespace App\Models;
+ 
+use Illuminate\Contracts\Auth\MustVerifyEmail;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+use Illuminate\Notifications\Notifiable;
+ 
+class User extends Authenticatable implements MustVerifyEmail
+{
+    use Notifiable;
+ 
+    // ...
+}
+```
+More info in: [Preparing model for email verification](https://laravel.com/docs/10.x/verification#model-preparation)
```

Comment:
Moved above. Same reason as below - text flows better. I agree this would have been the perfect place to put an "extras" link normally. But since this text is pretty long, it's just simpler to have all text in one place, I think.

```suggestion
```